src/app/content/styles/PopupStyles.tsx

```diff
@@ -18,6 +20,17 @@ export const popupBodyPadding = 2.4;
 export const headerHeight = 7.2;
 export const topBottomMargin = headerHeight + popupBodyPadding;
 
+const swapColors = ({colorSchema}: {colorSchema: BookWithOSWebData['theme']}) => {
+  const color = theme.color.primary[colorSchema].foreground;
+  return css`
+    color: ${color};
+
+    &:hover {
+      color: ${color === theme.color.text.white ? toolbarIconColor.base : toolbarIconColor.lighter};
+    }
```

Comment:
This is how it looks when hovered for Anatomy and Physiology:
![image](https://user-images.githubusercontent.com/31478212/90012964-a3dae380-dca4-11ea-93e6-cca8970c5209.png)